content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,77 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
 ```
 
 ## Description
 
-This command visualize difference between metrics among experiments in the
-repository history. Requires that Git is being used to version the metrics
-files.
-
-The metrics file needs to be specified through `-d`/`--datafile` option. Also, a
-plot can be customized with
-[plot templates](/doc/command-reference/plots#plot-templates) using the
-`--template` option. To learn more about the file formats and templates please
-see `dvc plots`.
+This command is a way to visualize the difference between metrics among
+experiments in the <abbr>repository</abbr> history. Requires that the target
+metric files are versioned with Git. These should specified with the `--targets`
+option.
 
 `revisions` are Git commit hashes, tag, or branch names. If none are specified,
-`dvc plots diff` compares metrics currently present in the
-<abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
-A single specified revision results in plotting the difference in metrics
-between the workspace and that version.
-
-In contrast to commands such as `git diff`, `dvc metrics diff` and
-`dvc params diff`, **any number of `revisions` can be provided**, and the
+`dvc plots diff` compares targets currently present in the
+<abbr>workspace</abbr> (uncommitted changes) with their latest committed
+versions (required). A single specified revision results in plotting the
```

Comment:
This part doesn't seem accurate. I've asked about it in https://discord.com/channels/485586884165107732/563406153334128681/719342575835676733 for now...

Reply:
OK it seems like it \*should\* be correct, but the current 1.0 version has this bug: iterative/dvc/issues/3974. Resolving.